packages/stores/src/stores/createListStore.ts

```diff
@@ -69,6 +72,43 @@ export const createListStore = <T extends object>(
         return false;
     };
 
+    const use = (...subscriptions: Plugin<T>[]): (() => void) => {
+        const unsubscribes = subscriptions.reduce<Array<() => void>>(
+            (acc, plugin) => {
+                const subscription = plugin(store);
+
+                if (subscription.delete) {
+                    acc.push(on('delete', subscription.delete));
+                }
+                if (subscription.dispose) {
+                    acc.push(on('dispose', subscription.dispose));
+                }
+                if (subscription.get) {
+                    acc.push(on('get', subscription.get));
+                }
+                if (subscription.insert) {
+                    acc.push(on('insert', subscription.insert));
+                }
+                if (subscription.keys) {
+                    acc.push(on('keys', subscription.keys));
+                }
+                if (subscription.reset) {
+                    acc.push(on('reset', subscription.reset));
+                }
+                if (subscription.set) {
+                    acc.push(on('set', subscription.set));
+                }
+
+                return acc;
+            },
+            [],
+        );
+
+        return () => {
+            unsubscribes.forEach((fn) => fn());
+        };
+    };
+
```

Comment:
I think because the `listStore` just wraps a normal store, you could just pass through the `use` of the backing store. 

``` diff
- const { state, onChange, reset, on } = store;
+ const { state, onChange, reset, on, use } = store;
```